src/panels/lovelace/entity-rows/hui-timer-entity-row.ts

```diff
@@ -40,6 +40,14 @@ class HuiTimerEntityRow extends LitElement {
     this._clearInterval();
   }
 
+  public connectedCallback(): void {
+    super.connectedCallback();
+    if (this._config!.entity) {
+      const stateObj = this.hass!.states[this._config!.entity];
+      this._startInterval(stateObj);
```

Comment:
Add a `console.log` to `startInterval`. I would not be surprised if we would now end up starting 2 intervals on initial adding it to the DOM

Reply:
`_startInterval` calls first clearInterval, so that's not going to happen.